.github/workflows/auto-merge-dependabot-prs.yml

```diff
@@ -19,13 +19,13 @@ jobs:
           github-token: "${{ secrets.GITHUB_TOKEN }}"
       - name: Wait other jobs are passed or failed
         if: ${{steps.metadata.outputs.update-type != 'version-update:semver-major'}}
-        uses: kachick/wait-other-jobs@v1.1.1
+        uses: kachick/wait-other-jobs@v1
         timeout-minutes: 30
         with:
           github-token: "${{ secrets.GITHUB_TOKEN }}"
-      - name: Enable auto approve and merge for Dependabot PRs
+      - name: Approve and merge
         if: ${{steps.metadata.outputs.update-type != 'version-update:semver-major'}}
-        run: gh pr review --approve "$PR_URL" && gh pr merge --auto --squash "$PR_URL"
+        run: gh pr review --approve "$PR_URL" && gh pr merge --auto --merge "$PR_URL"
```

Comment:
Currewntly merged main branch does not run CI. To check the behaviro with this change